src/BrazeMessage.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { DigitalSubscriberAppBanner } from './DigitalSubscriberAppBanner';
+
+type BrazeMessageProps = {
+    [key: string]: string | undefined;
+};
+
+type CommonComponentProps = {
+    onButtonClick: (buttonIndex: number) => void;
+    brazeMessageProps: BrazeMessageProps;
+};
+
+type ComponentMapping = {
+    [key: string]: React.FC<CommonComponentProps>;
+};
+
+const COMPONENT_MAPPINGS: ComponentMapping = {
+    DigitalSubscriberAppBanner,
+};
+
+export type Props = {
+    onButtonClick: (buttonIndex: number) => void;
+    componentName: string;
+    brazeMessageProps: {
+        [key: string]: string;
```

Comment:
What's the reason for not using the `BrazeMessageProps` type here?

Reply:
Good spot, I just missed this, no reason at all not to use it.